dev-environment.yml

```diff
@@ -28,11 +28,11 @@ dependencies:
   - numpy
   - pandas
   # Developer tools
+  - assertpy
```

Comment:
Do we need this package for this fix? I appreciate that it cleans up some of the test code for readability, however I would prefer to **not** add another dependency to scyjava.

Reply:
@elevans If it were a transitive dependency for the main codebase I'd fully agree, but since it's only a developer dependency... I think I feel less strongly than you do about it. :sweat_smile:

Reply:
Can I interest you in the power of readable test failure messages?

This is the original error message: 

```
assert abs(mb_total - mb_initial) < 5, f"{mb_total=} {mb_initial=}"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: mb_total=56 mb_initial=50
```

While this is th error message using assertpy:

```
AssertionError: [total mb should be close to initial] Expected <56> to be close to <50> within tolerance <5>, but was not.
```

The value becomes more clear when you write tests checking more complex things than number comparisons, take a scroll through their [README](https://github.com/assertpy/assertpy) to see if you recognize any patterns 😎 

A more concrete example is near the end of `test_arrays.py`:

```python
for i in range(len(nums_delta)):
    for j in range(len(nums_delta[i])):
        assert nums_delta[i][j] == pdoubles[i][j]
```

If you used assertpy you'd get a nice error message displaying both lists showing how they differ instead of just "false"

```python
for i in range(len(nums_delta)):
    assert_that(nums_delta[i]).is_equal_to(pdoubles[i])
```

Reply:
Persuasion successful! Curtis is right and perhaps I was too against adding a new dependency. Since this would just be in the main code base I have no problems here and in fact I think we could use `assertpy` in `pyimagej` and perhaps `imglyb`.